modules/cnv-creating-bridge-nad-cli.adoc

```diff
@@ -38,7 +38,6 @@ spec:
       {
         "type": "cnv-bridge", <2>
         "bridge": "br0", <3>
```

Comment:
I would drop the comma at the end. Not sure if the current state is invalid, but without comma it would be consistent with the other JSONs if nothing else.